### PR TITLE
John conroy/organ samples polish stories tests

### PR DESCRIPTION
**Pull request by john-conroy**
Tests/stories for #2324.

### CODE REVIEW

File: context/app/static/js/shared-styles/tables/SelectableRowCell/SelectableRowCell.jsx [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
-function SelectableRowCell({ rowKey, index, disabled }) {
+function SelectableRowCell({ rowKey, disabled }) {
   const { toggleRow, selectedRows, tableLabel } = useStore();
 
   return (
     <TableCell padding="checkbox">
       <Checkbox
         checked={selectedRows.has(rowKey)}
-        inputProps={{ 'aria-labelledby': `${tableLabel}-row-${index}-checkbox` }}
+        inputProps={{ 'aria-label': `${tableLabel}-row-${rowKey}-checkbox` }}
```
Comment (mccalluc) [inline context/app/static/js/shared-styles/tables/SelectableRowCell/SelectableRowCell.jsx:16]: If we're going to do this, should the label be something human readable?

File: context/app/static/js/shared-styles/tables/SelectableRowCell/SelectableRowCell.jsx [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
     </TableCell>
   );
 }
 
+SelectableRowCell.propTypes = {
+  /**
+   Unique key representing the table.
+  */
+  rowKey: PropTypes.string.isRequired,
+};
```
Comment (mccalluc) [inline context/app/static/js/shared-styles/tables/SelectableRowCell/SelectableRowCell.jsx:29]: I'm ok with not having proptypes... but if we do have them, should optional `disabled` be included?

File: context/app/static/js/shared-styles/tables/SelectableTableProvider/SelectableTableProvider.stories.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <TableBody>
+          {rowKeys.map((rowKey) => (
+            <TableRow key={rowKey}>
+              <SelectableRowCell rowKey={rowKey} />
+              <TableCell>{rowKey}</TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+      <Typography>You have selected rows {[...selectedRows].join(', ')}.</Typography>
```
Comment (mccalluc) [inline context/app/static/js/shared-styles/tables/SelectableTableProvider/SelectableTableProvider.stories.js:43]: `selecedRows` is a set? should this be sorted?

Comment (mccalluc) [reply context/app/static/js/shared-styles/tables/SelectableTableProvider/SelectableTableProvider.stories.js:43]: Oh, this is all just a test. Never mind.

**Review by mccalluc — APPROVED** [review]

File: context/app/static/js/shared-styles/tables/SelectableRowCell/SelectableRowCell.jsx (hunk 2 above)
Comment (john-conroy) [reply context/app/static/js/shared-styles/tables/SelectableRowCell/SelectableRowCell.jsx]: Added, thanks!

**Pull request merged**